testdata/ruby/transpile/balancedBrackets/Gemfile

```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
```

Comment:
Shouldn't this PR then not also update the other Go+Java repositories with Ruby implementations?